### DISCUSSION TITLE
My AI girl Jessica

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2319] [Show and tell]
Here are a few photos in a few weeks i have done, 
still learning, 0-10k followers on instagram in a month. wish a faster set up

![00177-2558544055](https://github.com/lllyasviel/Fooocus/assets/35565479/88689b2b-08cd-4402-96d6-1528abc3271c)

![00176-634405876](https://github.com/lllyasviel/Fooocus/assets/35565479/9a283f11-6849-48d0-9867-59f16dba2514)

![00175-634405875](https://github.com/lllyasviel/Fooocus/assets/35565479/59d2d896-37d9-4da8-8c00-abbc65958c87)

![00212-2388200351](https://github.com/lllyasviel/Fooocus/assets/35565479/008ce917-65b1-432c-91f1-c8f14396677f)

![00183-1957838851](https://github.com/lllyasviel/Fooocus/assets/35565479/2e6d75e1-9c02-4a63-bda6-f4d6e0c74074)
![00213-2388200352](https://github.com/lllyasviel/Fooocus/assets/35565479/92f5aeb6-3741-4ad7-b557-aa423346291c)
![00190-3108806814](https://github.com/lllyasviel/Fooocus/assets/35565479/d3c41668-911f-42cb-8d57-613c36243c64)
hub.com/lllyasviel/Fooocus/assets/35565479/57fca7c2-52a9-4ccc-b9fb-18dc8a310723)
![2024-01-31_18-13-55_9961](https://github.com/lllyasviel/Fooocus/assets/35565479/e6ec3741-b58a-46eb-8179-e35f92939f17)
![2024-01-31_14-37-02_6978](https://github.com/lllyasviel/Fooocus/assets/35565479/01cfe1b8-ed25-4a20-a1bc-545112996298)
![2024-01-31_14-00-14_1146](https://github.com/lllyasviel/Fooocus/assets/35565479/8e6b74be-f701-41e0-a2b0-58222a34da41)
![coffeewhitetophat](https://github.com/lllyasviel/Fooocus/assets/35565479/15492b06-9874-46d5-87e9-9eaa480f1ef2)

**Comment by mashb1t** [comment]
@lgwjames happy to connect with you in https://github.com/lllyasviel/Fooocus/discussions/2036 about how you did it!
How is the instagram profile called?

**Reply by lgwjames** [reply]
jessica_leigh2000

**Reply by monkrus** [reply]
Love it!

**Comment by eddyizm** [comment]
Great work!

**Comment by lgwjames** [comment]
thank you all for the comments. the only downside is being on a Apple Mac. it is slowing down my process because I have to wait 15 minutes to an hour for 4 photos and it is slowing me down doing some tweaking. I reached out to a guy on fiverr and has made me a lora from 35 photos which is helping a little bit. but I'm learning each day and I can't believe the response on Instagram even though I bet 80% are fake

**Reply by mashb1t** [reply]
Looks good, but, sorry to say, i have no doubts the 10k followers are botted. There is a drastic mismatch between likes (avg. 4 likes), interactions (no comments on any of the images) and follower count (10k+).

**Comment by lgwjames** [comment]
yep I agree, but hopefully after time some tru following

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by monkrus** [reply]
It might work as an initial push. Later on, it would be either paid adv or something really outstanding that the myriad of other AI models can`t do.

**Comment by lgwjames** [comment]
just up update, been slow as a lot of errors from this software over the last 3 months.
got a external rtx 3060 12gb to hook up and waiting on my holder to test 

few updated pics

![2024-05-27_18-59-55_8589](https://github.com/user-attachments/assets/848d55d9-5b53-4293-be1f-8b2b62664715)
![2024-05-27_15-13-37_5800](https://github.com/user-attachments/assets/f3cb7a1d-43fd-40e0-9489-21b8ed6f0879)
![2024-06-01_07-43-53_2112](https://github.com/user-attachments/assets/2184a09d-b8a5-4870-824d-937de8eaf4fc)
![2024-06-01_19-33-23_2855](https://github.com/user-attachments/assets/53836f45-156a-4e67-94da-111cbde2171d)

![2024-07-21_18-52-35_7621](https://github.com/user-attachments/assets/35093d75-2651-4fa0-bb17-bdcc743e7f88)

![2024-07-21_16-40-00_5895](https://github.com/user-attachments/assets/905c6263-8b2c-4f1c-9bc3-57ed37f0dce6)
fe302f5-6f2f-449b-b85d-4e0f20bc58d6)

**Comment by AbcITAndrzej** [comment]
How did you make it so you always have the same face generated? I mean the character.

**Reply by mashb1t** [reply]
https://github.com/lllyasviel/Fooocus/discussions/3390#discussioncomment-10172202

The power of wildcards/prompts!

**Reply by lgwjames** [reply]
i have removed that account from instragram. as that was sort of a learning curve, so now with a new set up made a new account

**Comment by lgwjames** [comment]
prompts and a trained lora from fiver

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by AbcITAndrzej** [comment]
ok, I know :-)

**Comment by lgwjames** [comment]
few more this week, still getting problems for last month of a lot of NON full body, head to toe images

Any ideas?

![2024-07-28_14-32-35_4670](https://github.com/user-attachments/assets/903e553c-4db3-42f8-b3a4-dd8031075413)
![2024-07-28_13-50-15_4220](https://github.com/user-attachments/assets/3c84155b-22ec-4435-81f0-440a532c0f8d)
![2024-07-28_13-35-52_4097](https://github.com/user-attachments/assets/8606b0b4-7d7d-4205-94f6-f76c5416a00a)
![2024-07-28_13-21-06_8457](https://github.com/user-attachments/assets/1a04104a-b579-42b3-af6c-bd0d419d6fcb)
![2024-07-28_13-06-29_3976](https://github.com/user-attachments/assets/76faeea8-1155-4817-b844-d4f873bc1204)

![2024-07-28_12-37-01_1413](https://github.com/user-attachments/assets/9abf44f6-ea99-4edc-9e29-565817de83c9)
![2024-07-28_12-22-37_5116](https://github.com/user-attachments/assets/77dd338b-957d-4841-b178-0c3c38f22444)
![2024-07-28_12-08-04_4329](https://github.com/user-attachments/assets/aae70e35-b98d-4542-97f6-8d5bf452f989)
![2024-07-28_11-53-31_5118](https://github.com/user-attachments/assets/a4b744a0-ee7f-4036-a3b8-20b37f4cb738)
![2024-07-28_11-38-57_7749](https://github.com/user-attachments/assets/3ce9cc89-6d3e-4a7d-9327-04327e924bb6)
![2024-07-28_11-24-05_4947](https://github.com/user-attachments/assets/576ac4c2-775e-4606-b7aa-17d54895a34d)
![2024-07-28_11-09-24_4361](https://github.com/user-attachments/assets/a2e8555e-940f-4084-8147-2c5082a5ad8f)
![2024-07-28_10-33-49_6580](https://github.com/user-attachments/assets/64670e94-db9d-467f-a9ea-85c37a3d9d7d)
![2024-07-28_10-18-54_8156](https://github.com/user-attachments/assets/6ef08c95-c9aa-4bab-a9b1-47aa0c94a19d)
![2024-07-28_09-16-27_5664](https://github.com/user-attachments/assets/ba5ba280-d76a-4d3b-a87e-8a40ecbb9830)
![2024-07-28_09-01-27_1565](https://github.com/user-attachments/assets/ebef5fbe-2be4-4edd-902f-24b9842cf2c6)
![2024-07-28_08-46-21_6768](https://github.com/user-attachments/assets/946d1ba8-0d14-4106-a6c2-766e76a5b981)

**Reply by hector-barriga** [reply]
Hey James, I'm a fan of your work. I need some advice on how to get different angles and outfits.
Could you shoot me an email? supreme.ai@gmail.com

**Comment by lgwjames** [comment]
would not email, what help are you after

**Reply by hector-barriga** [reply]
Since I'm new to image generation, any advice is valuable. In particular, 

1. How can I make the image full body, or the size I want? It usually makes an upper body portrait no matter the prompt instructions.
2. How can I create an image with a specific non-blurry background? Same struggle
3. I like the freckles work you have done. How did you get it so noticeable?
4. Do you know how to create images as talked from a smartphone or selfie?

Thanks a lot for any comments!

**Reply by lgwjames** [reply]
> Since I'm new to image generation, any advice is valuable. In particular,
> 
>     1. How can I make the image full body, or the size I want? It usually makes an upper body portrait no matter the prompt instructions.
> still have a few problems my self
>     2. How can I create an image with a specific non-blurry background? Same struggle
> 
>     3. I like the freckles work you have done. How did you get it so noticeable?
>  i just added in my wildcard file, light freckles around nose
>     4. Do you know how to create images as talked from a smartphone or selfie?
> you can download a model for that
> 
> Thanks a lot for any comments!
